tests/framework/utils.py

```diff
@@ -34,7 +35,12 @@ class CONST:
     # Paths
     REPO_ROOT = Path(__file__).absolute().parent.parent.parent
     HELM_CHART_ROOT = REPO_ROOT.joinpath("helm-chart")
-    DEFAULT_KIND_CONFIG = REPO_ROOT.joinpath("tests/framework/config/kind-config.yaml")
+
+    # Decide the config based on the environment
+    if os.getenv("BUILDKITE_ENV", default="") == "true":
```

Comment:
Why do we need a new kind config?

Reply:
The new part of the config is copied from https://github.com/ray-project/ray/pull/22035/files#diff-cbbac00da1880527c8b203aeb03a1d3c492610ee1fc97f868392589c5b831893.  I'm not sure about the technical details here, but it's necessary to get `kubectl` to work in buildkite.

I'm not sure if it has any ill effects for users who want to run the test locally, so to be safe I just made another config file.

Reply:
Got it. We can figure this out in the future.